charts/opentelmetry/README.md

```diff
@@ -0,0 +1,76 @@
+
+# Logzio-otel-k8s-metrics
+
+The Helm tool is used to manage packages of pre-configured Kubernetes resources that use Charts.
+logzio-otel-k8s-metrics allows you to ship metrics from your Kubernetes cluster with the OpenTelemetry collector.
+
+**Note:** This chart is a fork of the [opentelemtry-collector](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-collector) helm chart, it is also dependent on the [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics/tree/master/charts/kube-state-metrics) and [prometheus-node-exporter](https://github.com/helm/charts/tree/master/stable/prometheus-node-exporter) charts, that will be installed by default. To disable the dependency during installation, set `kubeStateMetrics.enabled` and `nodeExporter` to `false`.
+
+### Prerequisites:
+* [Helm 3](https://helm.sh/docs/intro/install/) installed
+
+
+### Deployment:
+
+#### 1. Add the logzio-otel-k8s-metrics repo to your Helm repo list
+
+```shell
+helm repo add logzio-otel https://logzio.github.io/logzio-helm/opentelemtry/
+```
+
+#### 2. Deploy
+
+
+#### Deploy with standard configuration:  
+
+Replace the Logz-io `<<METRICS-TOKEN>>` with the [token](https://app.logz.io/#/dashboard/settings/general) of the metrics account you want to ship to.
```

Comment:
Replace the Logz-io `<<PROMETHEUS-METRICS-SHIPPING-TOKEN>>` with the [token](https://app.logz.io/#/dashboard/settings/manage-tokens/data-shipping) of the metrics account you want to ship to.

Reply:
the link to lookup tokens has changed: https://app.logz.io/#/dashboard/settings/manage-tokens/data-shipping)